lib/Backend/GlobOpt.cpp

```diff
@@ -1060,6 +1060,9 @@ GlobOpt::MergePredBlocksValueMaps(BasicBlock *block)
         this->KillAllObjectTypes();
     }
 
+    this->blockData.capturedArgs = JitAnew(this->alloc, BVSparse<JitArenaAllocator>, this->alloc);
+    this->blockData.changedSyms = JitAnew(this->alloc, BVSparse<JitArenaAllocator>, this->alloc);
```

Comment:
This should go in InitBlockData().  The Clone and Reuse path will have valid BVs and you are allocating new ones unnecessarily.